hw/xbox/xid-sdl.c

```diff
@@ -60,6 +59,11 @@ enum {
     STR_SERIALNUMBER,
 };
 
+typedef enum HapticEmulationMode {
+    EMU_HAPTIC_LEFT_RIGHT,
+    EMU_UNSUPPORTED
```

Comment:
nit: I'd call this `EMU_NONE` or `EMU_DISABLED`, and put it first, so it's index 0.

If we ever allow user-overwrite for the mode (broken capabilities, or different modes supported) we don't have to rename stuff then. The suggested names are also still factually correct.